deploy/charts/firefly/templates/dataexchange/certificate.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and .Values.dataexchange.enabled .Values.dataexchange.certificate.enabled }}
+{{- if not (.Capabilities.APIVersions.Has "cert-manager.io/v1") }}
+# WARNING: cert-manager is not installed but certificate has been enabled, this will fail. Please either
+#          provide your own TLS certificate via tlsSecret.name or install cert-manager to resolve this.
+{{- end }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: "{{ include "firefly.fullname" . }}-dx"
+  labels:
+    {{- include "firefly.dataexchangeLabels" . | nindent 4 }}
+spec:
+  # NOTE: issuer should always be an internal / self-signed CA so that the subject is included
```

Comment:
TODO ref this issue in the comment https://github.com/jetstack/cert-manager/issues/3651